<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="taxonomer.iobio">
    <meta name="author" content="marthlab">
    <link rel="icon" href="../../favicon.ico">

    <title>taxonomer.iobio-faq</title>

    <link href="http://fonts.googleapis.com/css?family=Overlock+SC" rel="stylesheet" type="text/css">
    <!-- Bootstrap core CSS -->
    <link href="assets/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->    
    <link href="assets/css/site.css" rel="stylesheet">    

    <!-- Fonts -->
    <link href='http://fonts.googleapis.com/css?family=Quicksand:300,400' rel='stylesheet' type='text/css'>
    <link href='http://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
    <style>
        h2 {
            font-size: 50px;
        }

        .faq-question { font-size: 30px; margin-top: 45px;}
        .faq-answer { font-size: 16px; text-align: justify}
    </style>
  </head>

  <body>
    <div class="container">

      <div style="text-align:center;margin-bottom:20px" >
        <h2 style="font-weight:600;margin-top: -10px;"><span style="font-weight:600;color: #268bd2;">FAQ</span></h2>
      </div> 

      <div style="width:960px; margin:auto">
        <div style='text-align:center'>Google Chrome is the recommended browser for use with taxonomer.iobio.io</div>
        <div class='faq-question'>What is the difference between “Reads Sampled” and “Reads Classified”?</div>
        <div class='faq-answer'>
          <div>
            “Reads Samples” is the total number of reads analyzed by Taxonomer. The first step is analysis by Taxonomer’s Binner module, which compares every read to all reference databases in parallel, assigning reads to broad, non-exclusive taxonomic categories.
          </div>
          <br/>
          <div>
              “Reads Classified” refers to the subset of reads that were analyzed by Taxonomer’s Classifier and Protonomer modules. This includes reads derived from human mRNA, bacterial 16S rRNA, fungal ITS, and all viral/phage sequences.
          </div>
          <br/>
          <div >
            <img src='assets/images/buttons.png' style="float:left;height:190px"/>
            The complexity introduced by separate “binning” and “classification” results stem from two major sources: (1) not all reads identified as e.g. human or bacterial are used for classification purposes (see below) and (2) a strong sampling bias resulting in overrepresentation of scientifically or medically relevant viral, bacterial, and fungal taxa in reference databases resulting in lower sensitivity for classification of phylogenetically distant taxa. Keeping this in mind, reads sampled are shown below each bine (see example) and reads classified are shown in parentheses. For the ‘Human’ bin, reads sampled refers to all human reads, while reads in parentheses show mRNA-derived reads. With RNA-seq data, rRNA and mitochondrial reads account for most of the difference. For the ‘Bacterial’ bin, reads sampled refers to all bacterial reads while reads classified indicates the subset identified as 16S rRNA-derived. With RNA-seq data, usually >25% of bacterial reads are 16S rRNA-derived and are classified. For the ‘Viral’ bin, reads sampled refers to reads identified by the Binner module based on viral genome reference sequences. Reads classified indicates the number of ‘Viral’ and ‘Unknown’ reads identified to a viral taxon based on Taxonomer’s Protonomer module.
          </div>
          <br/>
          <div>
            Bin sizes are based on raw read counts and are not normalized to genome sizes. Thus, while they are generally indicative of, they do not necessarily accurately reflect, the biological composition of the sample at the cellular/viral particle level.
          </div>
        </div>

        <div class='faq-question'>How can I compare results between two or more samples?</div>
        <div class='faq-answer'>
          In its current version, Taxonomer does not provide functionality for comparing results between samples. However, users can either download raw results and perform comparisons in separate analyses (e.g. by using DESeq for transcript expression analyses) or simply open 2 or more windows simultaneously to visually compare results.        
        </div>

        <div class='faq-question'>How can I ensure rapid analysis if my internet connection has slow upload speeds?</div>
        <div class='faq-answer'>
          We recommend data storage at AWS S3 (https://aws.amazon.com/s3/) if upload speeds are limited. The test below was performed with the read 1 fastq file from SRR1553464 streamed from the SRA, from AWS, and from an iMac on a University of Utah Ethernet connection with ~900 Mbps upload speed.
          <div style="text-align:center">
            <img src='assets/images/Timing.jpg'/ style="height:250px">
          </div>
        </div>

        <div class='faq-question'>Can I analyze 16S amplicon data?</div>
        <div class='faq-answer'>
          Yes, performance of Taxonomer with 16S amplicon sequencing data is shown in the Taxonomer manuscript in Figure 2e
        </div>
 
        <div class='faq-question'>Why does the Display Threshold disappear after zooming in?</div>
        <div class='faq-answer'>
          The display threshold can only be adjusted when viewing all results from a given ‘bin’.
        </div>       



        <div class='faq-question'>Can I download Taxonomer results and what information is included?</div>
        <div class='faq-answer'>

          Yes, you can download the results by clicking on the 'Download' link: <br>

          <div style="text-align:center;">
           <img src='assets/images/download_results.png' style="width:220px;padding-top:5px;padding-bottom:5px;">
          </div>

          Taxonomer results can be downloaded in hierarchical JSON format or as tab-deliminted text file with read-level results that has the following columns: <br> 
          <div style="padding-left:10px;">
          1) Bin category <br>
          2) Read classified (C) or unclassified (U) following bin assigment <br>
          3) Read name <br>
          4) Taxid of read assignment.  For virus this corresponds to NCBI taxid.  For bacteria, this corresponds to an artificial ID to the greengenes database, likewise for fungi except to the UNITE database.  For Human this is an artificial ID to a gene.  <br>
          5) Internal sequence identifier of sequence that contributed either completely or equally (in case of a tie) to the read’s assignment.  <br>
          6) Number of taxonomic levels at which the read assignment was made (more levels is a deeper assignment).  Note this value cannot be compared between bin categories  because difference taxonomies are used — for example, one cannot use this to compare the depth of a bacterial assignment to that of a viral assignment. <br>
          7) Number of sequences tied for the assignment <br>
          8) Max sum of k-mer weights for read assignment <br>
          9) Standard deviation of sum of k-mer weights for read assignment <br>
          10) Read length <br>
          11) Number of k-mers used to make the read assignment <br>
          12) Bin placeholder (no external significance) <br>
        </div>


        </div>    


      </div>


    </div>
  </body>
</html>