<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="taxonomer.iobio">
    <meta name="author" content="marthlab">
    <link rel="icon" href="../../favicon.ico">

    <title>taxonomer.iobio Info</title>

    <!-- Bootstrap core CSS -->
    <link href="assets/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->    
    <link href="assets/css/site.css" rel="stylesheet">    

    <!-- Fonts -->
    <link href='http://fonts.googleapis.com/css?family=Quicksand:300,400' rel='stylesheet' type='text/css'>
    <link href='http://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
  </head>

  <body>
    <div class="container">

      <div style="text-align:center;" >
        <h2 style="margin-top: -10px;">taxonomer.iobio</h2>
        <h2 style="margin-bottom: 40px;margin-top: -10px;">Information</h2>
      </div> 

      <div>
        <a name="sample1"><h3>Human serum sample positive for Zaire ebolavirus</h3></a>
        <span style="text-align: justify">
        This will start analysis of the first ~40k reads of published RNA-Seq data of a human serum sample positive for Zaire ebolavirus from the current outbreak in Sierra Leone, West Africa.
        Selection of viral reads demonstrates ~75% of the viral reads being classified as Zaire Ebolavirus. Per published data, this sample had an ebolavirus load of 1.61E+09 copies/ml by qPCR.
        The associated publication is here: <a href="http://www.sciencemag.org/content/345/6202/1369.long">http://www.sciencemag.org/content/345/6202/1369.long</a>
        The data is from: <a href="http://www.ncbi.nlm.nih.gov/sra/?term=SRR1553464">http://www.ncbi.nlm.nih.gov/sra/?term=SRR1553464</a>
        </span>

        <div style="padding-top: 40px"></div>

        <a name="sample2"><h3>16S rRNA (V4) amplicon data from a stool sample of a child</h3></a>
        http://taxonomer.iobio.io/?url=https://s3.amazonaws.com/iobio/ERR501052_1.fastq
        16S rRNA (V4) amplicon data from a stool sample of a child enrolled in this study: <a href="http://www.nature.com/nature/journal/v510/n7505/full/nature13421.html">http://www.nature.com/nature/journal/v510/n7505/full/nature13421.html</a>
        Virtually all reads are binned as bacterial and this allows you to explore a sample with diverse bacterial flora.
        The data is from: <a href="http://www.ncbi.nlm.nih.gov/sra/?term=ERR501052">http://www.ncbi.nlm.nih.gov/sra/?term=ERR501052</a>

        <div style="padding-top: 40px"></div>
        <a name="sample3"><h3>Metagenomics study of a serum sample from a patient with hemorrhagic fever</h3></a>
        This is published data from a shotgun metagenomics study of a serum sample from a patient with hemorrhagic fever of unknown etiology (Democratic Republic of Congo). This data resulted in discovery of a novel rhabdovirus (Bas-Congo virus) as the cause of this patientâ€™s symptoms and a related outbreak <a href="http://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1002924">http://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1002924</a>
        Selection of the viral reads shows ~45% of reads being classified as Bas-Congo virus. The sample was contaminated with human rotavirus (see publication), which represents ~10% of the viral reads.
        The data is from: <a href="http://www.ncbi.nlm.nih.gov/sra/?term=SRR533978">http://www.ncbi.nlm.nih.gov/sra/?term=SRR533978</a>

      </div>
    </div>
  </body>
</html>